frontend/packages/ceph-storage-plugin/src/utils/clone-workflow.ts

```diff
@@ -0,0 +1,21 @@
+import { K8sKind, K8sResourceKind } from '@console/internal/module/k8s';
+
+import { KebabOption } from '@console/internal/components/utils';
+
+export const ClonePVC = (kind: K8sKind, resource: K8sResourceKind): KebabOption => {
+  return {
+    label: 'Clone',
+    callback: () => {
+      const pvcObject = { resource };
```

Comment:
This name is misleading. It's not a PVC object. It's props to the dialog. I would just inline the value below.